src/reducers/index.js

```diff
@@ -1,6 +1,11 @@
 import board from './board'
 import ship from './ships'
+import currentUser from './currentUser'
+import users from './users'
+import games from './games'
+import login from './login'
+import signup from './signup'
 
 export default {
-  board, ship
+  board, ship, currentUser, users, games, login, signup
```

Comment:
We need to put them line by line for more visibility and to easily solve conflits :smile: